src/library/Uno.Material/Styles/Controls/v2/RadioButton.xaml

```diff
@@ -1,203 +1,257 @@
+            <!--#endregion-->
+
+            <!--#region Fill-->
+            <StaticResource x:Key="RadioButtonOuterEllipseFill" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseFillDisabled" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFill" ResourceKey="PrimaryBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFillDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <!--#endregion-->
+
+            <!--#region HoverRing-->
+            <StaticResource x:Key="MaterialRadioButtonHoverRingFillPointerOver" ResourceKey="PrimaryHoverBrush" />
+            <StaticResource x:Key="MaterialRadioButtonHoverRingFillPressed" ResourceKey="PrimaryFocusedBrush" />
+            <!--#endregion-->
+
+            <!--#region Typo-->
+            <StaticResource x:Key="RadioButtonFontFamily" ResourceKey="MaterialRegularFontFamily" />
+            <StaticResource x:Key="RadioButtonFontWeight" ResourceKey="BodySmallFontWeight" />
+            <StaticResource x:Key="RadioButtonFontSize" ResourceKey="BodySmallFontSize" />
+            <StaticResource x:Key="RadioButtonCharacterSpacing" ResourceKey="BodySmallCharacterSpacing" />
+            <!--#endregion-->
+
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="Light">
+            <!--#region Foreground-->
+            <StaticResource x:Key="RadioButtonForeground" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonForegroundPointerOver" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonForegroundPressed" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonForegroundDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <!--#endregion-->
+
+            <!--#region Stroke-->
+            <StaticResource x:Key="RadioButtonOuterEllipseStroke" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseStrokeDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedStroke" ResourceKey="PrimaryBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedStrokeDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <!--#endregion-->
+
+            <!--#region Fill-->
+            <StaticResource x:Key="RadioButtonOuterEllipseFill" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseFillDisabled" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFill" ResourceKey="PrimaryBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFillDisabled" ResourceKey="OnBackgroundDisabledBrush" />
```

Comment:
Looking here: https://github.com/unoplatform/uno/blob/1ecff68b4c71dfd1f0f7df60fbbed9dd27ac1952/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/RadioButton_themeresources.xaml#L84-L99

We should be adding support for the Pressed and PointerOver visual states of the Stroke/CheckedStroke and Filled/CheckedFill property brushes

Reply:
I wasn't sure how to implement these missing ones as you pointed out, without using something like CombinedStates. What was messing with me was the following situation:

Let's say we have a checked RadioButton using RadioButtonOuterEllipseCheckedStroke, we then hover over that RadioButton with our mouse, so the stroke changes to RadioButtonOuterEllipseCheckedStroke**PointerOver**. Up until now, everything is fine. 

However, once we stop hovering over the RadioButton, it will return to normal state, and use that resource, which is RadioButtonOuterEllipseStroke (no **Checked**). So we find ourselves where the RadioButton is **still checked visually**, but the colour of the Stroke is from Normal state and not Checked state.

Reply:
I'll try following the way fluent does it [here](https://github.com/unoplatform/uno/blob/47c24ec94c48902a092bd01f4f006649743eead3/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/RadioButton_themeresources.xaml#LL366C34-L373C73). If I just have an OuterEllipse and an OuterEllipse_Checked, it should work.

Reply:
@kazo0 Done! Added the missing ones, I had to add an OuterEllipse_Checked and CheckEllipse_Unchecked which adds a bit of heaviness to the xaml compared to being able to use CombinedStates, but it was still a lot easier than I originally thought.

Reply:
I also updated the gif in the PR